src/styles/App.css

```diff
@@ -1,4 +1,301 @@
+}
+
+li {
+  list-style: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.p-100 {
+  padding: 0 100px;
+}
+
+.align-c {
+  align-items: center;
+}
+
+.just-c {
+  justify-content: center;
+}
+
+.just-sb {
+  justify-content: space-between;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.nav {
+  padding: 30px 100px;
+  background-color: #fff;
+  border-bottom: 2px solid var(--gray);
+  margin-bottom: calc(38px - 15px);
+}
+
+.title {
+  color: var(--azure);
+  font-size: 30px;
+  cursor: pointer;
+}
+
+.right-cont {
+  margin-left: 50px;
+}
+
+.book-header-title {
+  margin-right: 40px;
+  text-transform: uppercase;
+  font-size: 15px;
+  letter-spacing: 1.9px;
+  color: var(--black);
+  cursor: pointer;
+}
+
+.category-header-title {
+  color: var(--black);
+  opacity: 0.5;
+  font-size: 15px;
+  letter-spacing: 1.9px;
+  text-transform: uppercase;
+  margin-right: 10px;
+}
+
+#filterSelect {
+  cursor: pointer;
+}
+
+.prof {
+  width: 50px;
+  height: 50px;
+  border: solid 1px var(--gray);
+  border-radius: 50%;
+  position: absolute;
+  right: 100px;
+  bottom: 23px;
+  cursor: pointer;
+}
+
+.prof svg {
+  font-size: 1.25em;
+}
+
+.prof svg path {
+  fill: var(--azure);
+}
+
+.book-card {
+  font-family: 'Roboto Slab', sans-serif;
+  background-color: white;
+  margin: 15px 0;
+  padding: 32px 147px 26px 27px;
+  border: solid 1px var(--gray);
+  border-radius: 4px;
+  position: relative;
+}
+
+.book-cat {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 14px;
+  opacity: 0.5;
+  margin-bottom: 4px;
+}
+
+.book-title {
+  font-size: 22px;
+}
+
+.book-id {
+  font-weight: 300;
+  font-size: 14px;
+  color: var(--blue);
+  margin-bottom: 20px;
+}
+
+.lower-btns {
+  color: var(--blue);
+  font-size: 14px;
+  font-weight: 300;
+}
+
+.lower-btns p {
+  cursor: pointer;
+}
+
+.rmv-btn {
+  border: 0;
+  background-color: transparent;
+  font-family: 'Roboto Slab', sans-serif;
+  color: var(--blue);
+  font-size: 14px;
+  font-weight: 300;
+  cursor: pointer;
+}
+
+.btn-divider {
+  width: 2px;
+  height: 24px;
+  margin: 0 15px;
+  border: solid 1px var(--gray);
+}
+
+.progress {
+  position: absolute;
+  right: 40%;
+}
+
+.porcentage-oval {
+  background: linear-gradient(to bottom, var(--medium-blue), var(--dodger-blue));
+  display: inline-block;
+  width: 68px;
+  height: 68px;
+  border-radius: 50%;
+  margin-right: 20px;
+  padding: 5px;
+}
+
+.porcentage-oval span {
+  background: #fff;
+  display: block;
+  width: 58px;
+  height: 58px;
+  border-radius: 50%;
+}
+
+.porcentage {
+  font-size: 32px;
+  font-family: 'Montserrat', sans-serif;
+  color: var(--black);
+}
+
+.completed {
+  font-family: 'Montserrat', sans-serif;
+  color: var(--black);
+  opacity: 0.5;
+  font-size: 14px;
+}
+
+.divider {
+  width: 2px;
+  height: 70px;
+  border: 1px solid var(--gray);
+  position: absolute;
+  right: 400px;
+}
+
+.chapter-cont {
+  position: absolute;
+  right: 180px;
+}
+
+.current-ch {
+  font-size: 13px;
+  font-weight: 300;
```

Comment:
Let us do same as suggested earlier above for this style `font-weight: 300;` to avoid repetition
`fweight-3{ font-weight: 300;}`
so you can reuse it